SPARCore/GADArticle.h

```diff
@@ -1,22 +1,31 @@
 #import <Foundation/Foundation.h>
+#import "GADRemoteModel.h"
 
-@interface GADArticle : NSObject
+@interface GADArticle : GADRemoteModel
 
+@property (nonatomic, strong) NSDate *datePublished;
+@property (nonatomic, strong) NSString *brief;
+@property (nonatomic, strong) NSString *headerImage;
 @property (nonatomic, strong) NSString *publicationId;
+@property (nonatomic, strong) NSDate *dateEdited;
+@property (nonatomic, strong) NSString *articleId;
+@property (nonatomic, strong) NSString *title;
+@property (nonatomic, strong) NSString *content;
 @property (nonatomic, strong) NSArray *authors;
 @property (nonatomic, strong) NSURL *url;
-//@property (nonatomic, strong) NSInteger *pictureSize;
 @property (nonatomic, strong) NSString *series;
 @property (nonatomic, strong) NSArray *tags;
-@property (nonatomic, strong) NSString *brief;
-@property (nonatomic, strong) NSDate *datePublished;
-@property (nonatomic, strong) NSString *content;
-//@property (nonatomic, strong) NSSOMETHING *headerImage;
-@property (nonatomic, strong) NSString *title;
 @property (nonatomic, strong) NSString *issue;
-@property (nonatomic, strong) NSString *articleId;
+
++ (void) articlesFromPublication: (NSString *)publicationId
```

Comment:
How about `articlesForPublicationID:...` or something similar. In the future, we'll probably want to write a wrapper function which will take a (forthcoming) `GADPublication` object; that function should probably be called `articlesForPublication:...`.